src/ansys/fluent/core/launcher/fluent_container.py

```diff
@@ -184,38 +186,35 @@ def configure_container_dict(
     else:
         logger.debug(f"container_dict before processing: {container_dict}")
 
-    if not host_mount_path:
-        if file_transfer_service:
-            host_mount_path = pyfluent.USER_DATA_PATH
-        elif os.getenv("PYFLUENT_HOST_MOUNT_PATH", None):
-            host_mount_path = pyfluent.EXAMPLES_PATH
-        elif pyfluent.CONTAINER_MOUNT_PATH:
-            host_mount_path = pyfluent.CONTAINER_MOUNT_PATH
-        else:
-            host_mount_path = os.getcwd()
+    if not mount_source:
+        mount_source = os.getenv(
+            "PYFLUENT_CONTAINER_MOUNT_SOURCE", pyfluent.CONTAINER_MOUNT_SOURCE
+        )
+    if not mount_source:
+        mount_source = pyfluent.EXAMPLES_PATH if file_transfer_service else os.getcwd()
```

Comment:
```suggestion
    if not mount_source:
        if file_transfer_service:
             mount_source = pyfluent.USER_DATA_PATH
        elif os.getenv( "PYFLUENT_CONTAINER_MOUNT_SOURCE",  None):
            mount_source  = pyfluent.CONTAINER_MOUNT_SOURCE
         else:
             mount_source = os.getcwd()
```

Reply:
@hpohekar I'm not saying that your logic above is necessarily wrong but it look curious. Let's meet as a team to go over this.